src/metadataID3v2.ml

```diff
@@ -251,3 +248,239 @@ let parse_pic ?recode pic =
   let n = 5 + l + text_bytes in
   let pic_data = String.sub pic n (String.length pic - n) in
   { pic_format; pic_type; pic_description; pic_data }
+
+type frame_id =
```

Comment:
Is this type really useful (compared to simply passing strings for frame id...) ?